packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -104,6 +112,12 @@ class CodePushClient {
       body: json.encode(CreateAppCollaboratorRequest(email: email).toJson()),
     );
 
+    if (response.statusCode == HttpStatus.forbidden) {
```

Comment:
instead of handling the error here can we surface the CodePushForbiddenException with the server error and then return a custom error in the CLI since the error is CLI-specific?

Reply:
I'm not sure I follow – what do you mean "return a custom error in the CLI since the error is CLI-specific"? I think this message makes sense to any client that would be attempting to add a collaborator, no?

Reply:
I think the api client should ideally return what the server responds with and the CLI should be the one to interpret and override messages if we want to do that.

Reply:
If it makes sense for any client then we should update the server error imo.

Reply:
Ah gotcha, updated. PTAL